crates/parser/tests/snapshots/statements/valid/0043@8.snap

```diff
@@ -1,19 +1,40 @@
 ---
 source: crates/parser/tests/statement_parser_test.rs
-description: "/* TODO: CREATE TABLE films2 AS SELECT * FROM films; */ SELECT 1;"
+description: CREATE TABLE films2 AS SELECT * FROM films;
 ---
 Parse {
-    cst: SourceFile@0..64
-      CComment@0..55 "/* TODO: CREATE TABLE ..."
-      SelectStmt@55..64
-        Select@55..61 "SELECT"
-        Whitespace@61..62 " "
-        ResTarget@62..63
-          AConst@62..63
-            Iconst@62..63 "1"
-        Ascii59@63..64 ";"
+    cst: SourceFile@0..42
+      Create@0..6 "CREATE"
```

Comment:
It seems like the statement parser does not pick the root statement up. Can you check that?

Reply:
sure, let me try to investigate this one, but I will need some guidance for where to look

Reply:
fixed it 👍